src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue

```diff
@@ -165,7 +148,10 @@
 <script>
 
 import PollIcon from 'vue-material-design-icons/Poll.vue'
+import FileChartCheckOutline from 'vue-material-design-icons/FileChartCheckOutline.vue'
```

Comment:
A -> B -> .. -> `F` -> .. -> `P` -> .. -> Z

Reply:
I will look into eslint rules again

Reply:
Thank you for the alphabetical reminder 🤓